src/content/posts/2024/07/10/detecting-flaky-tests.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Tuist now supports detecting flaky tests"
+categories: ["Swift", "XcodeProj", "Xcode"]
```

Comment:
```suggestion
categories: ["Tests", "Swift", "XcodeProj", "Xcode"]
```

Would be good to always add a category to tie it with the feature the blog post is about (if any). Adding the `Tuist Projects` category doesn't make much sense as most need that, so I'd only use the categories for `Test`, `Cache`, and soon `Share`

Reply:
Good idea! I just noticed I've been using categories as tags, but they should be different things. Let's revisit this when we migrate to the new website. For now I'll add tests there.